test/cql-pytest/run.py

```diff
@@ -311,6 +311,8 @@ def run_scylla_cmd(pid, dir):
         # test/alternator/run.
         '--experimental-features=udf',
         '--experimental-features=keyspace-storage-options',
+        '--experimental-features=consistent-topology-changes',
+        '--experimental-features=tablets',
```

Comment:
This means test/cql-pytest/run will now **always** run Scylla with tablets enabled - just not used in the keyspace. I think this is fine at this stage of the development where consistent-topology-changes and tablets are stable "enough". We just need to be aware we did this. If you run with "vnode replication" it's not exactly the same as before this patch because of these two experimental flags.

Reply:
Yes, I think this should not cause a problem anymore. Especially not in a single-node test, this won't stress any of the new feature in consistent-topology-changes, so we are not exposed to any flakyness.